### ISSUE TITLE
Sflix [Couldn't find any video links]

### ISSUE THREAD

**Issue by HaZe5657** [jmir1/aniyomi-extensions#1026]
### Source information and language

Sflix 13.13 (English)

### Steps to reproduce

1. Play any movie or TV show

### Expected behavior

Short buffer followed by video playback.

### Actual behavior

Buffers for a while and shows error/pop-up saying "Couldn't find any video links."

![Screenshot_20221116-221543](https://user-images.githubusercontent.com/85792795/202242032-ed627806-c092-4be3-9e1a-86fdca666e96.png)

**I use the 1.0.0.1 cloudflare DNS, since a lot of the extensions, Sflix, SuperSteam, AnimePahe to name a few don't work properly without it.**

Viewing it under Webview shows that the site's been blocked by an IT Act, I live in India.

![Screenshot_20221116-151325](https://user-images.githubusercontent.com/85792795/202242126-6337314c-3cc5-46d9-9217-156f6cc267c9.png)

Loading up the sflix.to site in chrome however, it comes up fine.

I had experienced the same issue with Sflix before, however it was resolved after an update to the extension.

### Aniyomi version

App version: 0.12.3.8 (standard, dc8d1af, 72, 2022-08-25T23:30Z)

### Android version

Android version: 11 (SDK 30) Android build ID: ROQ31.166-63

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.5](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.jmir.xyz/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/jmir1/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by tsunaproxD** [comment]
I have the same problem

**Comment by HaZe5657** [comment]
> I have the same problem

Yeah, and it really sucks that the extension only works for so long after finally getting an update.
